dist/Data-Dumper/t/bless_var_method.t

```diff
@@ -1,9 +1,13 @@
 #!./perl -w
 # t/bless.t - Test Bless()
 
+use strict;
+use warnings;
+
+use Config;
+
 BEGIN {
     if ($ENV{PERL_CORE}){
-        require Config; import Config;
         no warnings 'once';
         if ($Config{'extensions'} !~ /\bData\/Dumper\b/) {
             print "1..0 # Skip: Data::Dumper was not built\n";
```

Comment:
This block could probably just be removed.  TEST (and hence harness) already skip the tests.